src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -57,7 +57,7 @@ export class TemplateActionService extends TemplateInstanceService {
     unhandledActions.forEach((action) => this.actionsQueue.push({ ...action, _triggeredBy }));
     const res = await this.processActionQueue();
     await this.handleActionsCallback([...unhandledActions], res);
-    if (!this.container.parent) {
+    if (!this.container?.parent) {
       await this.templateNavService.handleNavActionsFromChild(actions, this.container);
```

Comment:
Various catches (conditional chain `?`) are added in case actions are triggered from outside a template, where things like a container may not exist